sdk/maps/maps-render-rest/review/maps-render.api.md

```diff
@@ -459,10 +459,10 @@ export interface RenderGetMapStaticImageQueryParamProperties {
     center?: Array<number>;
     height?: number;
     language?: string;
-    layer?: "basic" | "hybrid" | "labels";
-    path?: string;
-    pins?: string;
-    style?: "main" | "dark";
+    path?: Array<string>;
+    pins?: Array<string>;
```

Comment:
@Lakicar95 
Previously, we added a directives _Change the MultiCollection Type to Array | String_ in [swagger/README.md](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/maps/maps-render-rest/swagger/README.md#change-the-multicollection-type-to-array--string) because the core-client-rest encode the TypeScript array to [csv collection](https://swagger.io/docs/specification/2-0/describing-parameters/#:~:text=csv%20(-,default,-)) as default. However, for the parameters `path` and `pin`, the API expects them to be multi collection.
```
API expects:
static/image?path={path1}&path={path2}&path={path3}

core-client-js encodes:
static/image?path={path1},{path2},{path3}
```
That's why we add the directive to change `path` and `pin` from `Array` to `string`, and add a util function [createMultiCollection](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/maps/maps-render-rest/src/createMultiCollection.ts) to transform.

Based on this context, you may want to check if core-client-rest supports multi collection by replacing the `createMultiCollection` with an array in `createPathQuery` and `createPinsQuery`:
```diff
export function createPathQuery(paths: Array<PolygonalPath | CircularPath>): string {
  // ...
-  return createMultiCollection("path", pathQueries);
+  return pathQueries;
}
```
If this works, it means the core-client-rest support multi collection and we can get rid of `createMultiCollection`
Otherwise, we should make change to directives _Change the MultiCollection Type to Array | String_ to change the `path` and `pin` back to `string`

Reply:
@andykao1213 Thanks a lot for the explanation. Seems it's not working just with pins `return pinsQueries,` but it works for `return pathQueries;`, and it was a problem with directive:

Previously it was like: where: $.paths["/map/static/{format}"].get.parameters[12] - pattern wasn't recognized

I will change it to comply with the doc, it should be like: where: $.paths["/map/static"].get.parameters[12] - without /{format}

Does it make sense to modify to go back to apply directive for both pin and paths? or just to keep it for pins (as for pins doesn't work)?

Reply:
Doesn't make sense that `pathQueries` works but not `pinsQueries`. I tried from my side and looks like the core-client-rest is not supporting multi-collection yet, this may be a bug from the API side. I think we should apply the directive to both pins and path to send the request in the correct shape

Reply:
Thanks Andrew, will do that! :)